src/accumulator/util.rs

```diff
@@ -391,6 +396,22 @@ mod tests {
 
     use bitcoin_hashes::sha256;
 
+    #[test]
```

Comment:
nit: Can you bring this test down in the file? There are some imports afterwards, I think it's clearer if imports come before any code.

Reply:
Addressed. I also moved up another import that was declared after a test code.